frontend/src/pages/modelServing/screens/projects/InferenceServiceModal/ManageInferenceServiceModal.tsx

```diff
@@ -49,6 +49,7 @@ const ManageInferenceServiceModal: React.FC<ManageInferenceServiceModalProps> =
   const [createData, setCreateData, resetData] = useCreateInferenceServiceObject(editInfo);
   const [actionInProgress, setActionInProgress] = React.useState(false);
   const [error, setError] = React.useState<Error | undefined>();
+  const isInferenceServiceNameWithinLimit = createData.name.trim().length <= 63;
```

Comment:
shouldn't be this checked with `translateDisplayNameForK8s`, since the k8s resource would add hyphens right?